Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php

```diff
@@ -315,7 +315,7 @@ private function getWrapperMethodSequence(Tokens $tokens, $method, $startIndex,
      *     - endIndex (int): The index of the function/method end.
      *     - bodyIndex (int): The index of the function/method body.
      *     - modifiers (array): The modifiers as array keys and their index as
-     *       the values, e.g. array(T_PUBLIC => 10).
+     *       the values, e.g. array(T_PUBLIC => 10)
```

Comment:
the thing is that due to Symfony standards you shouldn't have sentence here in the first places.
(btw, single line could have multiple sentences as well, not connected)